src/functions.rs

```diff
@@ -203,9 +203,9 @@ pub mod stdlib {
 
         /// The implementation of the standard [`child-index`][`crate::reference::functions#child-index`]
```

Comment:
This should also refer to `named-child-index`.

(I would have made a suggested change, but it's outside the lines that have been touched by this PR, so the interface doesn't allow me to.)

Reply:
Good catch!